10_files_lab/fileslab.txt

```diff
@@ -8,30 +8,67 @@ Online: https://www.tocode.co.il/bundles/unix1/lessons/fileslab
   - .highscore
 
 Write here the commands you used:
-
+>cd
+>mkdir lab
+>cd lab
+>touch a.out hero.c enemy.c game.c main.c human.h monster.h .highscore
+>ls -a 
 2. "cd" into lab folder and inside create the following subdirectories:
   - Music
   - Misc
   - Drivers
 
 Write here the commands you used:
-
+> mkdir Music Misc Drivers 
+> ls 
 3. Copy the file "/etc/passwd" to each subdirectory created in (2).
 
 Write here the commands you used:
+> cp /etc/passwd /home/amir/lab/Music
+> cp /etc/passwd /home/amir/lab/Misc
+> cp /etc/passwd /home/amir/lab/Drivers
 
 4. Create a file named myinfo and inside write your name and 3 things you like about Unix.
+> touch myinfo
+> vi myinfo
+	>"i"
+	> new for me to learn
+	> free
+	> can be confgured according to one's needs
+	>"esc" ":wq!"
+> cat myinfo
 Now create a new directory named "myinfo" and move the file into it.
 
-Did it work? Why? Can you create a file "myinfo" inside a directory named "myinfo" ? How ?
+Did it work? no
+Why? the directiry has already a file with the same name
+Can you create a file "myinfo" inside a directory named "myinfo" - yes
+How ?
+u need to create this directory in alter location
+copy the file into it
+copy entire directory to the needed location
+
+> cd /home/amir/
+> mkdir myinfo
+> cp /home/amir/lab/myinfo /home/amir/myinfo/
+> rm /home/amir/lab/myinfo
+> mv /home/amir/myinfo /home/amir/lab/
+
```

Comment:
you can use relative path to make things a bit easier:
```
~/  # can replace /home/amir
./ # can in that case also replace /home/amir (it means current dir)
```
so that:
```
mkdir myinfo
cp ./lab/myinfo ./myinfo/
rm ./lab/myinfo
mv ./myinfo ./lab/
```

can you think of a way to make this shorter? :)